package-lock.json

```diff
@@ -4906,7 +4900,15 @@
       "integrity": "sha1-MAV0OOrGz3+MR2fzhkjWaX11yQM=",
       "dev": true,
       "requires": {
-        "minimist": "0.2.1"
+        "minimist": "0.0.8"
```

Comment:
I don't think these changes should be in here

Reply:
https://github.com/radarlabs/radar-sdk-js/commit/f279dd75f0e9ddd946c4f08aa85a5c10e762c8e4